### ISSUE TITLE
pre containing br breaks compression

### ISSUE THREAD

**Issue by tomap** [kangax/html-minifier#879]
Compression will stop after encountering a br inside a pre.
Try this code with default options in the online version:
```
<pre>
<br>
</pre>
<u>
test fail
</u>
```
result is identical.

I looked at source code but could not figure out why

**Comment by alexlamsl** [comment]
Works for me using the online version:
```html
<pre>
<br>
</pre>
<u>
test fail</u>
```

**Comment by tomap** [comment]
That is not working!
If you replace the br with a ``<p></p>``
```
<pre>
<p></p>
</pre><u>test fail</u>
```
You can see that the rest of the code is on the same line as the ``</pre>``

which is not the case with a br

You can try with a larger example to be convinced:
```
<pre>
<br>
</pre>
<u>
test fail
<i>
tr
</i>
</u>
<br>
```

**Comment by tomap** [comment]
And on a much larger scale, you can check the source code of this page:
https://tpî.eu/2018/01/12/New-comment-system/

**Comment by tomap** [comment]
I did some more tests, and it seems any self closing tag inside a pre will trigger this issue (like img or else)

@alexlamsl Could you please reopen ?

**Comment by azopyros** [comment]
When using `<br>`, minification break

![capture d ecran 2018-01-29 a 15 30 07](https://user-images.githubusercontent.com/16599090/35515428-6c80c956-0509-11e8-9c4e-0527b0e91d6a.png)
![capture d ecran 2018-01-29 a 15 30 16](https://user-images.githubusercontent.com/16599090/35515430-6cc1d982-0509-11e8-943b-bef3872326c0.png)
![capture d ecran 2018-01-29 a 15 30 45](https://user-images.githubusercontent.com/16599090/35515431-6d0f397a-0509-11e8-8998-b871d8a12bca.png)

**Comment by tomap** [comment]
I believe this is now fixed by 51819e6ac3f11ebfd5d8b7d36dd9ae2ebddc0813
